app/code/Magento/Customer/view/adminhtml/templates/tab/cart.phtml

```diff
@@ -11,7 +11,7 @@
 <div class="content-header skip-header">
     <table>
         <tr>
-            <td><h4><?= $block->escapeHtml($block->getCartHeader()) ?></h4></td>
+            <td><h4><?= $escaper->escapeHtml($block->getCartHeader()) ?></h4></td>
```

Comment:
It seems like escaper is not defined in this file

Reply:
fixed